### DISCUSSION TITLE
How to make the virtual list component scroll more smoothly?

### DISCUSSION THREAD

**Discussion by ShangChien** [tusen-ai/naive-ui#5707] [Q&A]
I feel a slight difference between virtual lists and regular lists when scrolling：
- **this is slow speed gif for a virtual select：no smooth animate**
![scroll_hopping](https://github.com/tusen-ai/naive-ui/assets/48350636/11266fe0-d0ad-4ecd-a1c5-fe4297aff5c0)
- **this is slow speed gif for normal list smooth select：**
![scroll_smooth](https://github.com/tusen-ai/naive-ui/assets/48350636/1165f366-4d3c-4717-a650-0f7524690e8f)

#### I try following ways, but not work:

1. set style `scroll-behavior: smooth` to dom

2. config found in [vueuc](https://github.com/07akioni/vueuc/tree/81c90b9a8d34582bc4bc27039d10cc5821bdea90)

``` ts
export interface CommonScrollToOptions {
  behavior?: ScrollBehavior
  debounce?: boolean
}
```

How to make the virtual list component scroll more smoothly?
👀 anyone can help?
thank you !

**Comment by ShangChien** [comment]
MacOS behave well, this diff experience only show in windows OS.

**Comment by jahnli** [comment]
This issue does not have any recent activity. If you are still experiencing similar problems, open a new error, including a minimal copy of the problem
